canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.jsx

```diff
@@ -212,6 +227,13 @@ class DropDown extends React.Component {
 			"clear.selection": formatMessage(this.reactIntl, MESSAGE_KEYS.DROPDOWN_TOOLTIP_CLEARSELECTION)
 		};
 
+		const renderItem = (item) => (item ? (
```

Comment:
It's generally better to have this be it's own function.  Having it inline recreate the function x times.